packages/main/test/pages/Button.html

```diff
@@ -29,7 +29,8 @@
 </head>
 
 <body style="background-color: var(--sapBackgroundColor);">
-	<ui5-button icon="home"><!----><!----></ui5-button>
+	<ui5-button icon="home" id="icon-only-comment"><!----><!----></ui5-button>
+	<ui5-button icon="text" id="icon-only-blank-text"></ui5-button>
```

Comment:
Shouldn't this example rather be `> </ui5-button>` (have a single space inside)?